dev-python/immutabledict/immutabledict-2.2.3.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=poetry
+PYTHON_COMPAT=( python3_{9..11} )
+inherit distutils-r1
+
+DESCRIPTION="An immutable wrapper around dictionaries"
+HOMEPAGE="
+	https://github.com/corenting/immutabledict/
+	https://pypi.org/project/immutabledict/
+"
+SRC_URI="
+	https://github.com/corenting/immutabledict/archive/refs/tags/v${PV}.tar.gz
```

Comment:
No chance of using pypi tarball? Please ask upstream to include tests there.

Reply:
https://github.com/corenting/immutabledict/issues/196

Reply:
upstream released new version for us with tests in pypi tarball. I will bump the version.